tests/extensions/test_gunicorn.py

```diff
@@ -33,19 +33,20 @@ def input_yaml_fixture(monkeypatch, tmp_path):
     }
 
 
-def test_flask_extension(input_yaml, tmp_path):
-    applied = apply_extensions(tmp_path, input_yaml)
+def test_flask_extension(flask_input_yaml, tmp_path):
+    applied = apply_extensions(tmp_path, flask_input_yaml)
```

Comment:
Please combine this and the `test_django_extension` test below into a single parametrised test. Something like:

```suggestion
@pytest.mark.parametrize(
    ("input_yaml", "expected"),
    [
        (
            {
                "type": "charm",
                "name": "test-flask",
                "summary": "test summary",
                "description": "test description",
                "bases": [{"name": "ubuntu", "channel": "22.04"}],
                "extensions": ["flask-framework"],
            },
            {
                "actions": FlaskFramework._ACTIONS,
                ...  # The rest of the expected output here
            },
        )
    ],
)
def test_apply_extensions_correct(tmp_path, extension_class, input_yaml, expected):
    monkeypatch.setenv("CHARMCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS", "1")
    
    assert apply_extensions(tmp_path, flask_input_yaml) == expected
```

Reply:
You can make the `input_yaml` dict a constant and then have the fixture and this parametrization both return `FLASK_INPUT_YAML.copy()`

Reply:
Got it, I've created a helper function called `make_flask_input_yaml` and used it in the fixture. Thanks!